agent/src/main/java/com/code_intelligence/jazzer/api/MethodHook.java

```diff
@@ -143,6 +143,11 @@
    * The name of the class that contains the method that should be hooked,
    * as returned by {@link Class#getName()}.
    * <p>
+   * If an interface or abstract class is specified, also calls to all
+   * implementations and subclasses available during startup are hooked,
```

Comment:
Could we clarify `available during startup`? Perhaps `available on the classpath during startup`?

Reply:
Changed.